.gitignore

```diff
@@ -119,3 +119,6 @@ work_dirs/
 *.pth
 *.py~
 *.sh~
+
+# dataset
+data/
```

Comment:
line 107 already adds `./data/`, it should already work?

Reply:
I didn't noticed line 107, at least in my environment git (v2.11.0) complains and tell me `data` directory is part of untracked files.
It looks like the `./`has confused git in my case.
But removing initial `./` works for me! :smile:

Reply:
OK, we may keep both for now.